### ISSUE TITLE
decorators in code blocks are being treated as tags

### ISSUE THREAD

**Issue by Necroskillz** [TypeStrong/typedoc#187]
Given this doc:

``````
/**
 * Some Description
 *
 * ```
 * @RouteConfig([
 *   { path: '/about', component: AboutComponent, name: 'About' }
 * ])
 * ```
 */
``````

Output looks like this:
![image](https://cloud.githubusercontent.com/assets/346477/12790209/7a294f00-caa1-11e5-8285-5f431645132a.png)

**Comment by Necroskillz** [comment]
I fixed it for now by inserting invisible space after the @ sign.

http://www.fileformat.info/info/unicode/char/200B/browsertest.htm

**Comment by mattflix** [comment]
I encountered this behavior too.

The comment parser should not be looking for doc tags inside the code block maybe?

**Comment by aciccarello** [comment]
Related to #97

**Comment by aciccarello** [comment]
Should be fixed when #633 is released
